sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/LoadflowProvider.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.sensitivity_computation;
+
+import com.farao_community.farao.data.crac_api.NetworkElement;
+import com.powsybl.iidm.network.*;
+import com.powsybl.sensitivity.SensitivityFactor;
+import com.powsybl.sensitivity.SensitivityFunction;
+import com.powsybl.sensitivity.SensitivityVariable;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * To run a systematic sensitivity analysis and evaluate the flows in all states at once,
+ * hades requires sensitivities. We therefore extend RangeActionSensitivityProvider to use
+ * some of its conversion methods, and use a random PST from the network to create "dummy"
+ * sensitivities for each studied cnec.
+ *
+ * @author Philippe Edwards {@literal <philippe.edwards at rte-france.com>}
+ */
+public class LoadflowProvider extends RangeActionSensitivityProvider {
```

Comment:
shouldn't it be the opposite ? the RangeActionSensitivityProvider which extends the LoadFlowProvider.

As the List of RangeAction of the RangeActionSensitivityProvider is not used in this class.

Reply:
and functionnaly it seems strange to have access here to some methods of the RangeActionSensitivityProvider, such as : addSensitivityFactors(Set<RangeAction> rangeActions, Set<Cnec> cnecs)

Reply:
You're right, i switched the classes around